src/api/socket.ts

```diff
@@ -0,0 +1,70 @@
+import { io, Socket } from "socket.io-client";
+import backendApi from "./backend";
+
+const SOCKET_URL = 'ws://' + backendApi.BACKEND_URL.split('//')[1];
```

Comment:
Overall great work :rocket: 

Can you add an if here that works like this?

if `import.meta.env.VITE_APP_PROD` is 0, then use `ws`. Otherwise, use `wss`.